test/integration/golden_response/get_statvar_group_node/school_race.json

```diff
@@ -1,59 +1,86 @@
 {
   "absoluteName": "Person With Enrollment Level = Enrolled in College Undergraduate Years, Race",
-  "childStatVarGroups": [
+  "childStatVars": [
```

Comment:
@pradh This seems to be a big change. Can you confirm?

Reply:
This is because of cl/410075652